test-utils/src/main/scala/com/mesosphere/utils/mesos/MesosTest.scala

```diff
@@ -31,14 +31,23 @@ import scala.util.Try
   * @param imageProviders Comma-separated list of supported image providers, e.g., APPC,DOCKER.
   * @param seccompConfigDir
   * @param seccompProfileName
+  * @param executorRegistrationTimeout Defaults to one minute.
+  * @param fetcherStallTimeout Defaults to one minute and must be lower or equal to the executor registration timeout.
   */
 case class MesosAgentConfig(
     launcher: String = "posix",
     containerizers: String = "mesos",
     isolation: Option[String] = None,
     imageProviders: Option[String] = None,
     seccompConfigDir: Option[String] = None,
-    seccompProfileName: Option[String] = None) {
+    seccompProfileName: Option[String] = None,
+    executorRegistrationTimeout: Option[Duration] = None,
```

Comment:
Where are `executorRegistrationTimeout` and `fetcherStallTimeout` set in tests? Maybe instead of `Option[Duration] = None` use `Duration = 1.seconds` directly?

Reply:
The Jenkins Plugin uses them. The defaults are from Mesos. I think you are correct. `Option` is just confusing here.